<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>imageViewerWindow</class>
  <widget class="QWidget" name="imageViewerWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>569</width>
        <height>480</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>ImageViewer</string>
    </property>
    <property name="styleSheet">
      <string notr="true">#toolbarFrame QPushButton
      {
	      background: #193D66;
      }
      </string>
    </property>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
      <property name="spacing">
        <number>0</number>
      </property>
      <property name="leftMargin">
        <number>0</number>
      </property>
      <property name="topMargin">
        <number>0</number>
      </property>
      <property name="rightMargin">
        <number>0</number>
      </property>
      <property name="bottomMargin">
        <number>0</number>
      </property>
      <item>
        <widget class="QWidget" name="workspaces">
          <layout class="QHBoxLayout" name="workspacesHorizontalLayout">
            <!--
              <item>
              <widget class="QPushButton" name="imageViewerButton">
              <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
                <string/>
              </property>
              <property name="icon">
                <iconset>
                  <normaloff>../../../src/CADVision-demo/Images/open.png</normaloff>../../../src/CADVision-demo/Images/open.png</iconset>
              </property>
              <property name="flat">
                <bool>true</bool>
              </property>
              <property name="iconsize" stdset="0">
                <size>
                  <width>93</width>
                  <height>27</height>
                </size>
              </property>
            </widget>
            </item>
            <item>
              <spacer name="horizontalSpacer">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
            -->
            <item>
              <widget class="QFrame" name="toolbarFrame">
                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>48</height>
                  </size>
                </property>
                <property name="maximumSize">
                  <size>
                    <width>50</width>
                    <height>200</height>
                  </size>
                </property>
                <layout class="QVBoxLayout" name="toolBarVerticalLayout">
                  <property name="spacing">
                    <number>5</number>
                  </property>
                  <property name="leftMargin">
                    <number>0</number>
                  </property>
                  <property name="topMargin">
                    <number>7</number>
                  </property>
                  <property name="rightMargin">
                    <number>0</number>
                  </property>
                  <property name="bottomMargin">
                    <number>7</number>
                  </property>
                  <!--
                  <item>
                    <spacer name="leftHorizontalSpacer">
                      <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                        <size>
                          <width>40</width>
                          <height>20</height>
                        </size>
                      </property>
                    </spacer>
                  </item>
                  -->
                  <item>
                    <widget class="QPushButton" name="openImgButton">
                      <property name="toolTip">
                        <string>Extract image from file</string>
                      </property>
                      <property name="text">
                        <string/>
                      </property>
                      <property name="icon">
                        <iconset resource="icons.qrc">
                          <normaloff>:/icons/Images/opne-image.png</normaloff>
                          <normalon>:/icons/Images/open-image.png</normalon>:/icons/Images/open-image.png</iconset>
                      </property>
                      <property name="iconSize">
                        <size>
                          <width>24</width>
                          <height>24</height>
                        </size>
                      </property>
                      <property name="checked">
                        <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                        <bool>false</bool>
                      </property>
                      <property name="default">
                        <bool>false</bool>
                      </property>
                      <property name="flat">
                        <bool>false</bool>
                      </property>
                      <property name="miminumSize" stdset="0">
                        <size>
                          <width>60</width>
                          <height>34</height>
                        </size>
                      </property>
                      <property name="maxinumSize" stdset="0">
                        <size>
                          <width>167721</width>
                          <height>34</height>
                        </size>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="zoomOutButton">
                      <property name="toolTip">
                        <string>Zoom Out</string>
                      </property>
                      <property name="text">
                        <string/>
                      </property>
                      <property name="icon">
                        <iconset resource="icons.qrc">
                          <normaloff>:/icons/Images/zoom-out.png</normaloff>
                          <normalon>:/icons/Images/zoom-out.png</normalon>:/icons/Images/zoom-out.png</iconset>
                      </property>
                      <property name="iconSize">
                        <size>
                          <width>24</width>
                          <height>24</height>
                        </size>
                      </property>
                      <property name="checked">
                        <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                        <bool>false</bool>
                      </property>
                      <property name="default">
                        <bool>false</bool>
                      </property>
                      <property name="flat">
                        <bool>false</bool>
                      </property>
                      <property name="miminumSize" stdset="0">
                        <size>
                          <width>60</width>
                          <height>34</height>
                        </size>
                      </property>
                      <property name="maxinumSize" stdset="0">
                        <size>
                          <width>167721</width>
                          <height>34</height>
                        </size>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="zoomInButton">
                      <property name="toolTip">
                        <string>Zoom In</string>
                      </property>
                      <property name="text">
                        <string/>
                      </property>
                      <property name="icon">
                        <iconset resource="icons.qrc">
                          <normaloff>:/icons/Images/zoom-in.png</normaloff>
                          <normalon>:/icons/Images/zoom-in.png</normalon>:/icons/Images/zoom-in.png</iconset>
                      </property>
                      <property name="iconSize">
                        <size>
                          <width>24</width>
                          <height>24</height>
                        </size>
                      </property>
                      <property name="checked">
                        <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                        <bool>true</bool>
                      </property>
                      <property name="default">
                        <bool>false</bool>
                      </property>
                      <property name="flat">
                        <bool>false</bool>
                      </property>
                      <property name="miminumSize" stdset="0">
                        <size>
                          <width>60</width>
                          <height>34</height>
                        </size>
                      </property>
                      <property name="maxinumSize" stdset="0">
                        <size>
                          <width>167721</width>
                          <height>34</height>
                        </size>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="FitWindowButton">
                      <property name="toolTip">
                        <string>Zoom In</string>
                      </property>
                      <property name="text">
                        <string/>
                      </property>
                      <property name="icon">
                        <iconset resource="icons.qrc">
                          <normaloff>:/icons/Images/fit-to-window.png</normaloff>
                          <normalon>:/icons/Images/fit-to-window.png</normalon>:/icons/Images/fit-to-window.png</iconset>
                      </property>
                      <property name="iconSize">
                        <size>
                          <width>24</width>
                          <height>24</height>
                        </size>
                      </property>
                      <property name="checked">
                        <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                        <bool>true</bool>
                      </property>
                      <property name="default">
                        <bool>false</bool>
                      </property>
                      <property name="flat">
                        <bool>false</bool>
                      </property>
                      <property name="miminumSize" stdset="0">
                        <size>
                          <width>60</width>
                          <height>34</height>
                        </size>
                      </property>
                      <property name="maxinumSize" stdset="0">
                        <size>
                          <width>167721</width>
                          <height>34</height>
                        </size>
                      </property>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>
            <item>
              <widget class="QScrollArea" name="scrollArea">
                <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                </property>
                <property name="widgetResizable">
                  <bool>false</bool>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>`
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                    <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>560</width>
                      <height>440</height>
                    </rect>
                  </property>
                  <widget class="QLabel" name="imageLabel">
                    <property name="geometry">
                      <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>560</width>
                        <height>440</height>
                      </rect>
                    </property>
                    <property name="minimumSize">
                      <size>
                        <width>0</width>
                        <height>0</height>
                      </size>
                    </property>
                    <property name="maximumSize">
                      <size>
                        <width>16777215</width>
                        <height>877</height>
                      </size>
                    </property>
                    <property name="text">
                      <string/>
                    </property>
                    <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                    </property>
                  </widget>
                </widget>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <resources>
    <include location="icons.qrc"/>
  </resources>
  <connections/>
</ui>
